### DISCUSSION TITLE
VS Code setup

### DISCUSSION THREAD

**Discussion by thbeu** [jenstroeger/python-package-template#382] [Q&A]
Is it desired to have a default settings file for VS code or is the repository purely meant for workflow automation purposes?

**Comment by jenstroeger** [comment]
@thbeu good point, I actually have a work item here to add VSCode defaults! Would you like to open a PR, otherwise I can add them today.

Addendum: I opened issue https://github.com/jenstroeger/python-package-template/issues/383

**Comment by thbeu** [comment]
I am happy to review.

**Comment by jenstroeger** [comment]
> I am happy to review.

PR https://github.com/jenstroeger/python-package-template/pull/386. Thank you!

**Comment by jenstroeger** [comment]
Shipped with [v2.5.0](https://github.com/jenstroeger/python-package-template/releases/tag/v2.5.0), thank you for your contribution @thbeu 🤓
